Util/imosCompile.m

```diff
@@ -44,9 +44,17 @@ function imosCompile()
 
 % assume we are running from snapshot/export (see
 % snapshot/buildBinaries.py)
-exportRoot    = pwd;
-toolboxRoot   = [exportRoot filesep '..' filesep '..'];
-stagingRoot   = [exportRoot filesep '..' filesep 'staging'];
+if nargin < 1,
+  exportRoot    = pwd;
+  toolboxRoot   = [exportRoot filesep '..' filesep '..'];
+  stagingRoot   = [exportRoot filesep '..' filesep 'staging'];
+elseif nargin == 1,
+  exportRoot = toolboxRoot;
+  stagingRoot = [ toolboxRoot '/' 'dist'];
```

Comment:
Use `filesep` instead of '/' so that it is platform independent.

Another way to do it is `stagingRoot = fullfile(toolboxRoot, 'dist');`

Reply:
amended